genai-perf/README.md

```diff
@@ -758,6 +758,10 @@ second. For example, 'request_latency:300' or
 'output_token_throughput_per_user:600'. Multiple key:value pairs can be
 provided, separated by spaces. (default: `None`)
 
+##### `-override-config`
```

Comment:
Thanks for adding this! We have templates as well, so the template needs to be updated too. You can just copy and paste these lines to README_template.

Reply:
This was also added to the `README_template` does it need to go somewhere else?

Reply:
I missed it earlier. Nice job!